### PR TITLE
chore(core): fix warning in spell check 

### PR DESCRIPTION
**Pull request by thevaibhav-dixit**

### CODE REVIEW

File: core/api/src/domain/quiz/config.ts [hunk 1]
```diff
@@ -11,10 +11,10 @@ export const QuizzesValue: Record<QuizQuestionId, Satoshis> = {
   moneyEvolution: 2 as Satoshis,
   whyStonesShellGold: 3 as Satoshis,
   moneyIsImportant: 3 as Satoshis,
-  moneyImportantGovernement: 4 as Satoshis,
+  moneyImportantGovernment: 4 as Satoshis,
```
Comment (nicolasburtey) [inline core/api/src/domain/quiz/config.ts:14]: I don't think this should be changed. this string is also recorded in the db. 
I think we should add an exemption for this word instead

**Review by nicolasburtey — CHANGES_REQUESTED** [review]
it's better to not change `moneyImportantGovernment` otherwise we'll need to do migration. let's add that to the "skip list"

File: core/api/src/domain/quiz/config.ts [hunk 2]
```diff
@@ -11,10 +11,10 @@ export const QuizzesValue: Record<QuizQuestionId, Satoshis> = {
...
   WhatIsFiat: 2 as Satoshis,
   whyCareAboutFiatMoney: 2 as Satoshis,
-  GovernementCanPrintMoney: 2 as Satoshis,
+  GovernmentCanPrintMoney: 2 as Satoshis,
```
Comment (thevaibhav-dixit) [inline core/api/src/domain/quiz/config.ts:17]: So we also leave this as is ? @nicolasburtey

Comment (nicolasburtey) [reply core/api/src/domain/quiz/config.ts:17]: yes

**Review by nicolasburtey — CHANGES_REQUESTED** [review]
still need to revert the changes on core/api/src/domain/quiz/config.ts and core/api/src/domain/quiz/sections.ts

File: typos.toml [hunk 3]
```diff
@@ -23,3 +25,5 @@ default.extend-ignore-re = ["\"eyJ[a-zA-Z0-9_\\-\\.]+\""]
 Ons = "Ons"
 Lsat = "Lsat"
 quizs = "quizs"
+Aggrement = "Aggrement"
```
Comment (bodymindarts) [inline typos.toml:28]: I don't think we should allow this typo across the whole code base when it is only present in 1 file. Can we make a narrower exclude?

**Review by bodymindarts — APPROVED** [review]

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**